### DISCUSSION TITLE
Bluetooth heaset mic don't work

### DISCUSSION THREAD

**Discussion by Enigma1309** [blueman-project/blueman#2333] [Q&A] [answered]
tl;dr 
Whenever an application uses my mic, my audio profiles gets switched automatically, but my mic don't work
but if i switch profile (a2dp to hsp/hfp) manually it stars working. i've been searching for the solution for past month but i can't find a fix...

---

My headphone - boAt Rockerz 510
here's `pactl list card`
![image](https://github.com/blueman-project/blueman/assets/94917588/ed3213c2-2fe4-497f-9b58-a70c74d91d98)

i've found that i have these two input devices in pauvcontrol
![image](https://github.com/blueman-project/blueman/assets/94917588/db5f74e5-e66e-49aa-9d49-da1bd0553c67)

The device with monitor in start (Monitor of boAt Rockerz 510), is mistaken for microphone by all apps, whenever an apps chooses it (automatically) , the currently playback audio (say a yt video playing in background)  gets picked into the specific app as a mic input.

but if i manually switch the input device to boAt Rockerz (without the Monitor in initial name), the automatic profile switching takes place in my system.

the even after the profile gets switched, the audio from mic get's picked by the application...

---

Is there any way to stop the audio from the output to get routed to the monitor of headset..
or is there any way i can set my default mic to Headset...(not the monitor)

Note:- this may be device specific issue, but any help is appreciated 
thank you

**Comment by cschramm** [comment]
Well, the green checkmark in pavucontrol is what sets the default input device. What you're saying is that even though the first device is set as default, any application will use the monitor device (or even set it as default?)? It looks like you're using PipeWire which used to have all sorts of trouble with remembering defaults. Did you check its logs yet?

**Reply by Enigma1309** [reply]
idk how to check logs any help would be appreciated

**Reply by cschramm** [reply]
Probably `journalctl --user -u pipewire.service`.

**Reply by Enigma1309** [reply, marked as answer]
idk how but the 8-9 months old problem got fixed yesterday, now everything works fine
i've switched various distro and checked their live boot if there's a problem , every distro had it.... 
but amazingly yesterday it started working out of the blue,
BTW thanks for showing interest in the discussion <3
